utils/content.ts

```diff
@@ -17,14 +17,16 @@ const articlesDirectory = path.join(process.cwd(), 'content', 'articles');
 export type IContent = {
   title: string;
   slug: string;
-  basePath: string
+  basePath: string;
   date: Date;
   id: string;
-  draft?: boolean
+  draft?: boolean;
 
-  selectedWork?: boolean
+  selectedWork?: boolean;
   description?: string;
-  previewImage?: string
+  previewImage?: string;
+  tags?: string[];
+  category?: string;
```

Comment:
I think for this type, the optional parameters for `tags` and `category` are missing. I logged the `content` object and I think these parameters are missing.

```
{
  title: 'Markdown Test',
  date: '2020-01-02',
  slug: 'test',
  description: 'Designing and building a website to connect quality photographers in Lagos, Nigeria',
  category: 'bug-fixes',
  tags: [ 'html', 'css', 'javascript' ]
}
```